server/tserver/src/main/java/org/apache/accumulo/tserver/log/LogSorter.java

```
@@ -247,6 +247,26 @@ void writeBuffer(String destPath, List<Pair<LogFileKey,LogFileValue>> buffer, in
     }
   }
 
+  // faster replacement for String.format("part-r-%05d.rf", part);
+  private static final String suffix = ".rf";
+
+  private String getFilename(int part) {
+    String prefix;
+    if (part < 10) {
```

Can part be negative? Might want to throw an exception in that case.